custom.js

```diff
@@ -0,0 +1,35 @@
+// Author: Mickaël Canouil
+// Version: <1.0.0>
+// Description: Change image src depending on body class (quarto-light or quarto-dark)
+// License: MIT
+function updateImageSrc() {
+    var bodyClass = window.document.body.classList;
+    var images = window.document.getElementsByTagName('img');
+    for (var i = 0; i < images.length; i++) {
+      var image = images[i];
+      var src = image.src;
+      var newSrc = src;
+      if (bodyClass.contains('quarto-light') && src.includes('_dark')) {
```

Comment:
@maelle. I just changed the names here because I am not using the same images names.

Reply:
ok!